--- ../dl/Squeak-4.10.2.2593-src/./unix/vm/sqUnixMain.c	2011-01-24 02:31:12.000000000 +0100
+++ ./unix/vm/sqUnixMain.c	2016-01-31 16:32:16.000000000 +0100
@@ -892,6 +892,9 @@
     }
 }
 
+extern SqModuleName(display, null);
+extern SqModuleName(sound, null);
+
 static void loadModules(void)
 {
   loadImplicit(&displayModule, "DISPLAY",     "display", "X11");
@@ -905,18 +908,21 @@
 #	 if defined(DEBUG_MODULES)
 	  fprintf(stderr, "squeak: %s driver defaulting to vm-%s-%s\n", md->type, md->type, md->name)
 #	 endif
-	    ;
+                  ;
   }
 
+
   if (!displayModule)
     {
-      fprintf(stderr, "squeak: could not find any display driver\n");
-      abort();
+      displayModule = SqModuleName(display, null);
+      //fprintf(stderr, "squeak: could not find any display driver\n");
+      //abort();
     }
   if (!soundModule)
     {
-      fprintf(stderr, "squeak: could not find any sound driver\n");
-      abort();
+      soundModule = SqModuleName(sound, null);
+      //fprintf(stderr, "squeak: could not find any sound driver\n");
+      //abort();
     }
 
   dpy= (struct SqDisplay *)displayModule->makeInterface();
